discord/user.py

```diff
@@ -143,6 +146,11 @@ def _update(self, data: UserPayload) -> None:
         self._banner = data.get("banner", None)
         self._accent_colour = data.get("accent_color", None)
         self._avatar_decoration = data.get("avatar_decoration_data", None)
+        nameplate = (data.get("collectibles") or {}).get("nameplate", None)
```

Comment:
instead of that ``(... or {})`` couldnt you just do
```suggestion
        nameplate = data.get("collectibles", {}).get("nameplate", None)
```
?

Reply:
No bc `data["collectibles"]` can be set but null

https://discord.com/developers/docs/resources/user#user-object